Automattic/newspack-plugin
# Update footer branding

### All Submissions:


### Changes proposed in this Pull Request:

This PR replaces the Newspack logo in the footer with new combined WordPress.com and Google News Initiative branding, as has been mocked up [here](https://cloudup.com/iytXwR9SSHQ).

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. View the initial setup Wizard (WP Admin > Settings > Newspack > Setup Wizard) and confirm the footer logo has been switched from the small Newspack icon to the WordPress.com and Google News Initiative logos, in white:

![image](https://user-images.githubusercontent.com/177561/88198344-75fc0500-cbf8-11ea-8e0a-19342867250c.png)

3. On the Newspack Dashboard, confirm that the new logo is displaying in grey:

![image](https://user-images.githubusercontent.com/177561/88198460-9035e300-cbf8-11ea-806f-c9fccd39f098.png)

4. On one of the wizard subpages, confirm that the new logo is displaying in grey: 

![image](https://user-images.githubusercontent.com/177561/88198502-9deb6880-cbf8-11ea-8e81-44069100ac02.png)

### Other information:
